sky/skylet/ray_patches/cli.py.patch

```diff
@@ -0,0 +1,11 @@
+0a1,4
+> # Adapted from https://github.com/ray-project/ray/blob/master/dashboard/modules/job/cli.py
```

Comment:
Nit: This is our own fix, so "adapted from" -> "From"? Maybe change `master/` to `releases/1.13.0` too as in other patches?